isis/scripts/downloadIsisData

```diff
@@ -189,33 +184,24 @@ def main(mission, dest, cfg_path, parsedArgs, kwargs):
         quit(-1)
     log.debug(f"Remote Sources: {config_sources}")
 
-    supported_missions = OrderedDict({})
-    for source in sorted(config_sources, key=lambda x: x.split("_")[-1]):
-        parsed_name = source.split("_")
-        # If it is a mission, it should be in the format <mission_nam>_<source_type>
-        if len(parsed_name) == 2 and parsed_name[1] in ["usgs:", "naifKernels:"]:
-            remotes_mission_name = parsed_name[0]
-            supported_missions[remotes_mission_name] = supported_missions.get(remotes_mission_name, []) + [source]
+    supported_missions = []
+    for source in config_sources:
+        if not any(x in source for x in ["asc_s3:", "esa:", "naif:", "jaxa:", "sbn:", "usgs:", "naifKernels:", "legacybase"]):
```

Comment:
`["usgs", "naifKernels"]` can probably be removed from the list. 

I think including legacy base in this is also causing it to not be downloaded. Seems to be downloading base directly rather than downloading the legacybase target to the base folder.

Reply:
I wanted to keep the supported_missions list to be only the names of the mission and not include `<mission>_usgs` or `<mission_naifKernels>` purely for user entry. I believe there is an error later down that displays this list to the user and I don't want any confusion. I will look into the legacybase issue.

Reply:
Just had it look for `_` instead since there are multiple with underscores that we don't want to add to the list. Removed `legacybase` since we want users to know that is an option. Removed it from list when user chooses `all`.